Translations/pt_BR.po

```diff
@@ -938,7 +938,7 @@ msgstr "Cajado de {:s}"
 #: Source/control.cpp:691 Source/control.cpp:1143
 msgid "{:d} Charge"
 msgid_plural "{:d} Charges"
-msgstr[0] "{:d} cargas"
+msgstr[0] "{:d} carga"
```

Comment:
Good

Reply:
Do you plan to write this for all 500 changes :D

Reply:
> Still some 50 comments letf, if you guys can thumbs up the ones you agree with then I will merge them. Also there are a handful of comments with multiple or no suggestion, it would be a big help if you could make it very clear what change I should merge in thease cases 😁

The ones I didn't comment is because I agree or I can live with. I will try to address the ones with multiple suggestions. Thanks again @AJenbo

Reply:
It's hard for me to tell if you agree or simply didn't see a comment, or maybe remain undecided. So since I'm just a clueless mediator in this case it would be helpful if you guys could be very explicit :)